.github/scripts/handle-mochawesome-report.js

```diff
@@ -83,9 +83,13 @@ function formatSummary(report) {
 
   let summary = "### Failed Tests Summary\n\n";
 
+  let isHeader = true;
```

Comment:
Should it be like this?

![image](https://github.com/user-attachments/assets/ad9d5222-af76-4117-8d3a-1890500c5dbc)

Reply:
yeah, I understand that the styling is different, I was thinking about splitting this table into two tables (table per file) or making title bold